### PR TITLE
Fix XA Datasource test

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
@jmazzitelli  could you please review and eventually merge?

File: hawkular-agent-itest-parent/hawkular-agent-itest-dist/src/main/resources/wildfly/patches/standalone.xsl [hunk 1]
```diff
@@ -938,6 +938,14 @@
...
         </resource-type-set-dmr>
 
+        <resource-type-set-dmr name="XA Datasource" enabled="true">
+          <resource-type-dmr name="XA Datasource"
+                             resourceNameTemplate="XA Datasource [%-]"
+                             path="/subsystem=datasources/xa-data-source=*"
+                             parents="WildFly Server"
+                             metricSets="Datasource Pool Metrics,Datasource JDBC Metrics" />
+        </resource-type-set-dmr>
+
```
Comment (jmazzitelli) [inline hawkular-agent-itest-parent/hawkular-agent-itest-dist/src/main/resources/wildfly/patches/standalone.xsl:948]: should we put this same definition in the "real" standalone.xsl in kettle?

File: hawkular-wildfly-monitor/src/main/java/org/hawkular/agent/monitor/log/AgentLoggers.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+package org.hawkular.agent.monitor.log;
+
+import org.jboss.logging.Logger;
+
+/**
+ * @author <a href="https://github.com/ppalaga">Peter Palaga</a>
+ */
+public final class AgentLoggers {
+
+    public static MsgLogger getRestLogger(Class<?> clazz) {
```
Comment (jmazzitelli) [inline hawkular-wildfly-monitor/src/main/java/org/hawkular/agent/monitor/log/AgentLoggers.java]: what does "getRestLogger" mean? This really doesn't involve REST does it?

Comment (ppalaga) [inline hawkular-wildfly-monitor/src/main/java/org/hawkular/agent/monitor/log/AgentLoggers.java]: Good catch. This was a blind copy-paste from Metrics. Will fix.

**Comment by ppalaga** [comment]
@jmazzitelli let's speak about this before merging

**Comment by jmazzitelli** [comment]
uh oh, moving away from junit to testng - some people aren't going to be happy :)

**Comment by ppalaga** [comment]
> uh oh, moving away from junit to testng - some people aren't going to be happy :)

But some will maybe? /me is becoming an official pupil of @jsanda ;)

**Pull request merged**